packages/adapter-vercel/index.d.ts

```diff
@@ -9,7 +9,7 @@ export interface ServerlessConfig {
 	 * @default Same as the build environment
 	 * @deprecated
 	 */
-	runtime?: `nodejs${number}.x`;
+	runtime?: `nodejs${number}.x` | `bun${string}`;
```

Comment:
Probably should be `bun${number
```suggestion
	runtime?: `nodejs${number}.x` | `bun${number}.x`;
```